lte/gateway/c/oai/include/ngap_messages_types.h

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/****************************************************************************
+  Source      ngap_messages_types.h
+  Date        2020/07/28
+  Subsystem   Access and Mobility Management Function
+  Author      Ashish Prajapati
+  Description Defines NG Application Protocol Messages
+
+*****************************************************************************/
+#pragma once
+
+#include "3gpp_24.008.h"
+#include "3gpp_38.401.h"
+#include "3gpp_38.413.h"
+#include "3gpp_38.331.h"
+#include "3gpp_23.003.h"
+#include "TrackingAreaIdentity.h"
+#include "Ngap_Cause.h"
```

Comment:
Where does this file come from?

Reply:
@ashish-pr , you need to add the following change to make it compile for `mme_oai`

```
diff --git a/lte/gateway/c/oai/tasks/udp/CMakeLists.txt b/lte/gateway/c/oai/tasks/udp/CMakeLists.txt
index 8ef59fda4..38d05336a 100644
--- a/lte/gateway/c/oai/tasks/udp/CMakeLists.txt
+++ b/lte/gateway/c/oai/tasks/udp/CMakeLists.txt
@@ -7,6 +7,8 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../../lib/bstr)
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../../lib/hashtable)

 set(S1AP_C_DIR ${PROJECT_BINARY_DIR}/tasks/s1ap/r15)
+set(NGAP_C_DIR ${PROJECT_BINARY_DIR}/tasks/ngap/r16)
 include_directories(${S1AP_C_DIR})
+include_directories(${NGAP_C_DIR})

 add_library(TASK_UDP udp_primitives_server.c)
 ```

Reply:
Hi @ssanadhya , for me its compiling successfully in dev-VM .
I am not getting error with respect to Ngap_Cause.h.
make FEATURES=mme_oai build_oai, compiled code without any errors

Reply:
> Where does this file come from?

Its part of auto-generated files

Reply:
@ashish-pr , as mentioned over Slack, please try `make clean` and then do `make FEATURES=mme_oai build_oai` to reproduce the issue.